polyfill/test/datetime.mjs

```diff
@@ -209,6 +209,9 @@ describe('DateTime', () => {
       it('balance', () => equal(`${new DateTime(2019, 1, 32, 0, 0, 0, 0, 0, 0, 'balance')}`, '2019-02-01T00:00'));
       it('throw when bad disambiguation', () =>
         throws(() => new DateTime(2019, 1, 1, 0, 0, 0, 0, 0, 0, 'xyz'), TypeError));
+      it('reject leap second', () => throws(() => new DateTime(2016, 12, 31, 23, 59, 60, 0, 0, 0, 'reject'), RangeError));
```

Comment:
I assume this and similar tests will be updated to use `.from()` once #260 is resolved?

Reply:
Yes, indeed.